src/tools/rust-analyzer/crates/ide-db/src/generated/lints.rs

```diff
@@ -301,7 +301,7 @@ pub const DEFAULT_LINTS: &[Lint] = &[
     },
     Lint {
         label: "let_underscore_drop",
-        description: r##"non-binding let on a type that implements `Drop`"##,
+        description: r##"non-binding let on a type that has a destructor"##,
```

Comment:
Actually: please revert this change; afaict it's auto-generated, and rust-analyzer will deal with it on their own.